modules/seqexec/engine/src/main/scala/seqexec/engine/UserEvent.scala

```diff
@@ -19,7 +19,7 @@ sealed trait UserEvent[+F[_], S, U] extends Product with Serializable {
 
 object UserEvent {
 
-  final case class Start[S, U](id: Observation.Id, user: Option[UserDetails], clientId: ClientId, userCheck: S => Boolean) extends UserEvent[Nothing, S, U]
+  final case class Start[S, U](id: Observation.Id, user: Option[UserDetails], clientId: ClientId) extends UserEvent[Nothing, S, U]
```

Comment:
The `userCheck` was removed because the resources check is not done inside `Engine` anymore.

Reply:
Where is it done then?

Reply:
In `SeqexecEngine`. I already had to do something similar in the implementation of `SeqexecEngine.startFrom`.